Added marking add-ons as deprecated (#949)
replicatedhq/kurl.sh: src/markdown-pages/add-ons/docker.md

weight: 36
title: "Docker Add-On"
addOn: "docker"
isDeprecated: true
---
## Deprecation Notice

### This add-on is deprecated.

> As of March 27, 2023, kURL no longer intends to continue to offer this add-on as part of the ongoing kURL project. This add-on is now considered deprecated, and may no longer be offered by the project after September 31st, 2023. Existing installs that use this add-on will be best effort supported during this deprecation window. Kubenetes 1.24.0+ no loner supports Dockershim, therefore you must use an alternative CRI such as [containerd](/docs/add-ons/containerd).  Please remove Docker from your spec and replace it with [Containerd](https://kurl.sh/docs/add-ons/containerd) to migrate existing installs.

Reviewer: Why? The March 27th date marks the beginning of our deprecation period.

weight: 36
title: "Docker Add-On"
addOn: "docker"
isDeprecated: true
---
## Deprecation Notice

### This add-on is deprecated.

> kURL will not be moving forward with support of this add-on in kURL starting March 23, 2023 and will be fully deprecated after September 31st 2023.  Existing installs that use this add-on will continue to be best-effort supported until the deprecation date.  To migrate to [Containerd](https://kurl.sh/docs/add-ons/containerd) please change your kURL spec to containerd to migrate existing installs.

Reviewer 2: ```suggestion
kURL will not be moving forward with support of this add-on starting March 23, 2023 and will be fully removed after September 31st 2023.  Existing installs that use this add-on will continue to be best-effort supported until the removal date.  Please remove Docker from your spec and replace it with [Containerd](https://kurl.sh/docs/add-ons/containerd) to migrate existing installs.
```

## Summary

Docker is a CRI (Container Runtime Interface).
If Docker is not used, an alternative CRI must be used in its place.
See [containerd documentation](/docs/add-ons/containerd) for more information.

> Kubenetes 1.24.0+ does not support Dockershim.  As a result of this,  therefore you must use an alternative CRI, such as [containerd](/docs/add-ons/containerd), instead.

Reviewer 2: ```suggestion
Kubenetes 1.24.0+ no loner supports Dockershim, therefore you must use an alternative CRI such as [containerd](/docs/add-ons/containerd).
```
